driver-core/src/main/com/mongodb/MongoClientSettings.java

```diff
@@ -184,12 +184,10 @@ public DnsClient getDnsClient() {
     }
 
     /**
-     * Gets the {@link InetAddressResolver} to use for looking up the {@link java.net.InetAddress} instances for each host.
-     *
-     * <p>If set, it will be used to look up the {@link java.net.InetAddress} for each host, via
-     * {@link InetAddressResolver#lookupByName(String)}. Otherwise, {@link java.net.InetAddress#getAllByName(String)} will be used.
```

Comment:
This was incorrect, as "Otherwise, ..." did not take into account `InetAddressResolverProvider`. `Builder#inetAddressResolver` explains the behavior in full, so I added `@see Builder#inetAddressResolver(InetAddressResolver)`.